docs/utilities.rst

```diff
@@ -14,8 +14,8 @@ The following commands are provided:
 
 * :ref:`mof_compiler`
 
-  A MOF compiler that takes MOF files as input and creates, updates or
-  removes CIM instances, classes or qualifier types in a CIM repository.
+  A MOF compiler that takes MOF files as input and updates the CIM repository
+  of a WBEM server with the result.
```

Comment:
My one argument with this is that the CIM repository may or may not be part of a server.  It may also be a standalone repository for other usage such as documentation.

Reply:
Karl, what you say is true for the MOF compiler API, but this section is about the mof_compiler utility command and as I understand it, it supports only the CIM repository in a WBEM server, and no user-pluggable repository like the API does. Please double check.

Reply:
You are right but even that is too limited.  It really just sends messages to create/delete classes/instances/qualifierdeclarations in a WEMServer defined by the script input parameters.

Normally this is the cim repository but it could also be a provider.

Reply:
But what you said is sufficient.

Reply:
Ok, then I consider the discussion resolved and the PR approved.